portal/client/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Do you need any of CSS from this file? it was auto generated by CRA

Reply:
Will check and update.